elements/howto-tabs/howto-tabs.js

```diff
@@ -284,24 +311,81 @@
       // changes its role to `tab`.
       this.setAttribute('role', 'tab');
       if (!this.id)
-        this.id = `howto-tabs-tab-generated-${dashTabCounter++}`;
+        this.id = `howto-tabs-tab-generated-${howtoTabCounter++}`;
+
+      // Set a well-defined initial state.
+      this.selected = false;
+      this.setAttribute('aria-selected', 'false');
+      this.setAttribute('tabindex', -1);
+    }
+
+    /**
+     * Properties and their corresponding attributes should mirror one another.
+     * To this effect, the property setter for `selected` handles truthy/falsy
+     * values and reflects those to the state of the attribute. It’s important
+     * to note that there are no side effects taking place in the property
+     * setter. For example, the setter does not set `aria-selected`. Instead,
+     * that work happens in the `attributeChangedCallback`. As a general rule,
+     * make property setters very dumb, and if setting a property or attribute
+     * should cause a side effect (like setting a corresponding ARIA attribute)
+     * do that work in the `attributeChangedCallback`. This will avoid having to
+     * manage complex attribute/property reentrancy scenarios.
+     */
+    attributeChangedCallback() {
+      const value = this.hasAttribute('selected');
+      this.setAttribute('aria-selected', value);
+      this.setAttribute('tabindex', value?0:-1);
```

Comment:
would you mind spacing these so it's ` this.setAttribute('tabindex', value ? 0 : -1);`